extensions/guacamole-auth-header/src/main/java/org/apache/guacamole/auth/header/AuthenticationProviderService.java

```diff
@@ -71,6 +71,9 @@ public AuthenticatedUser authenticateUser(Credentials credentials)
 
             // Get the username from the header configured in guacamole.properties
             String username = request.getHeader(confService.getHttpAuthHeader());
+            
+            //write username to the credentials object to make tokenfilter work
```

Comment:
Please follow the existing code style. Compare:

>     //write username to the credentials object to make tokenfilter work

vs. other comments in incubator-guacamole-client:

>     // Get the username from the header configured in guacamole.properties

Relative to the established style, this comment is missing the space following `//`, and should be in sentence case (first letter capitalized).